pkg/recipe/variable.go

```diff
@@ -350,6 +350,17 @@ func (vv *VariableValues) UnmarshalYAML(unmarshal func(interface{}) error) error
 					Rows:    rows,
 				}
 			}
+		// Multiselect values without a value are an empty []interface{} and we want to cast them into MultiSelectValues
+		case []interface{}:
+			multiV := make(MultiSelectValue, 0, len(v))
+			for _, interfaceV := range v {
+				str, ok := interfaceV.(string)
+				if !ok {
+					return fmt.Errorf("failed to cast []interface{} value into a MultiSelectValue ([]string): %v", v)
```

Comment:
Mostly likely these errors will be printed to the user, so the error message needs to be less technical. Like "failed to read value to multi-select value..."